src/oscontainer.py

```diff
@@ -117,6 +119,15 @@ def oscontainer_build(containers_storage, src, ref, image_name_and_tag,
         print("Copying ostree commit into container: {} ...".format(rev))
         run_verbose(["ostree", "--repo=" + dest_repo, "pull-local", src, rev])
 
+        # Copy commitmeta.json in to the oscontainer at /
+        commitmeta_source = os.path.join(
+            os.getcwd(), 'builds', ostree_version, 'commitmeta.json')
+        commitmeta_dest = os.path.join(mnt, 'commitmeta.json')
+        print("Writing {} to os-container root".format(commitmeta_source))
+        with open(commitmeta_source, 'r') as inf:
+            with open(commitmeta_dest, 'w') as outf:
+                outf.write(inf.read())
```

Comment:
Why not just `shutil.copyfile()`?

Reply:
Habit. But I don't mind changing it :-) :+1:

Reply:
Updated.